recirq/qml_lfe/syc_bell_pair.txt

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
the file extensions for these really should be `.json`

Reply:
Changed.